cmd/updater/update.sh

```diff
@@ -249,7 +254,39 @@ TEMPDIR=""
 TARFILE=""
 UPDATESRCDIR=""
 
+function download_tools_update() {
+    SPECIFIC_VERSION=$1
+    echo "downloading tools update ${SPECIFIC_VERSION}"
+    TEMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t "tmp")
+    export CLEANUP_UPDATE_TEMP_DIR=${TEMPDIR}
+    trap "rm -rf ${CLEANUP_UPDATE_TEMP_DIR}" 0
+
+    TARFILE=${TEMPDIR}/${LATEST}.tar.gz
+    UPDATESRCDIR=${TEMPDIR}/a
+    mkdir "${UPDATESRCDIR}"
```

Comment:
The lines above looks wrong, `TARFILE` is used later in the script to extract the archive. I suspect that the tools will not be extracted as expected. It looks like you copied the code below without checking how all the variables are used later in the script.

Reply:
It does work correctly due to the order of operation, but I renamed the tools tar file var to avoid confusion.